crates/forge-runner/src/build_trace_data.rs

```diff
@@ -192,6 +198,14 @@ fn build_profiler_call_type(value: CallType) -> ProfilerCallType {
     }
 }
 
+fn build_profiler_trace_entry(value: &TraceEntry) -> ProfilerTraceEntry {
```

Comment:
unidiomatic construction (use `From` instead)

Reply:
I think both types here do not belong to this package so unfortunately you cannot. It is consistent with other methods in this module though.